README.md

```diff
@@ -1,50 +1,71 @@
-MythXvsc is an extension for running MythX smart contract analysis from your favourite IDE.
+MythXvsc is an extension for running MythX smart contract analysis from Visual Studio Code.
 
 The extensions provides:
 
--   Smart contract compilation (via `solidity` VSCode extension)
--   Log in to MythX platform
+-   Smart contract compilation (via the Solidity VS Code extension)
+-   Authentication to the MythX platform
 -   AST extraction from compiled source
 -   Submission of analysis
+-   Displaying analysis result in VS Code in a linting fashion
 
 ## Instructions
 
-After installing the extension please enter your registered MythX `ethAddress` and `password` in VSCode user settings. These fields are properties of `mythxvsc` as shown in the screenshots below:
+# Access token login (preferred)
```

Comment:
This should be the way we tell users to Authenticate. Should remove "(preferred)" and make it such that this is the only authentication option. Authenticating w/ Eth address/Password should not be listed as an authentication method, rather an option that is listed as deprecated.